string directionR
int8 velocityR
string directionL
int8 velocityL
---
